messages/teleporter/message_handler.go

```diff
@@ -183,47 +183,63 @@ func (m *messageHandler) ShouldSendMessage(destinationClient vms.DestinationClie
 		return false, nil
 	}
 
-	var decision bool = true
+	deciderRejectedMsg, err := m.deciderRejectedMessage()
+	if err != nil {
+		m.logger.Warn(
+			"Error delegating to decider",
+			zap.String("warpMessageID", m.unsignedMessage.ID().String()),
+			zap.String("teleporterMessageID", teleporterMessageID.String()),
+		)
+	}
+	if deciderRejectedMsg {
+		m.logger.Info(
+			"Decider rejected message",
+			zap.String("warpMessageID", m.unsignedMessage.ID().String()),
+			zap.String("teleporterMessageID", teleporterMessageID.String()),
+			zap.String("destinationBlockchainID", destinationBlockchainID.String()),
+		)
+	}
+
+	return true, nil
```

Comment:
If we rename the method to `deciderAcceptedMessage`, then we can emit a log if the returned bool is false, and directly return the variable.

Reply:
see https://github.com/ava-labs/awm-relayer/pull/344#discussion_r1690052917